pkg/liqoctl/install/utils.go

```diff
@@ -128,11 +128,11 @@ func installOrUpdate(ctx context.Context, helmClient *helm.HelmClient, k provide
 }
 
 func generateValues(chartValues, commonValues, providerValues map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
A great addition would be a test that taking different values maps as input, verifies that the output is the expected one.